src/frontend.md

```diff
@@ -5,31 +5,55 @@
 ```
 src/
   components/ (can be dumb and smart)
-    bankAccount/
+    [All components should be placed in some entity folder or shared folder]
+    bankAccount/ (entity, module, .etc)
       BankAccountTable/
         [no nested directories]
+        [avoid index.ts]
+        [other files should be placed in queries or services]
         BankAccountTable.ts
         BankAccountTable.test.ts
         BankAccountTableBadge.ts (child component)
+        useBankAccountTableBadge.ts (ui hook used only in the current component)
         BankAccountTable.styled.ts
+        useBankAccountTable.ts (ui hook used only in the current component)
+        icon.svg|png|jpeg... (the icon or any static file used only in the current component)
     shared/
       SomeNonEntityComponent/
+      media/ (shared static files)
+        [no nested directories]
+        icon.svg|png|jpeg...
   pages/
     bankAccount/
       BankAccountList/
+        [no nested directories]
+        [other files should be placed in queries or services]
+        [components for the page should be placed in components folder]
         BankAccountList.page.ts
         BankAccountList.page.test.ts
```

Comment:
how about `.spec` here? for integration tests

Reply:
added